application/views/index.php

```diff
@@ -50,12 +30,28 @@
             <div class="col-lg-8">
                 <?php
                 $blog = new blog();
-                $result = $blog->getBlog(1);
+                $result = [];
+                if (isset($_GET['idCategory'])) {
+                    $blog_post_category_id = $_GET['idCategory'];
+                    $blog_post_category_obj = new blog_post_categories();
+                    $resultIdCategory = $blog_post_category_obj->displayBlogpostId($blog_post_category_id);
+                    // xdebug($resultIdCategory);
+
+                    $result = $blog->findById($resultIdCategory);
+                    //xdebug($resultByCat);
+
+                } else {
+                    $resultIdCategory = 1;
+                    $result = $blog->getBlogByUser($resultIdCategory);
```

Comment:
pag hindi naprovide ang idCategory, findall mo nlng kahit sinong user nun. para kung sakali 3 ang contributors ng blog mo, irereturn lahat ng articles nila

Reply:
ahh, noted po kuys nyan hehe. thank you ulit